cmake/onnxruntime_unittests.cmake

```diff
@@ -1367,6 +1367,14 @@ if (NOT onnxruntime_BUILD_WEBASSEMBLY)
             LIBS ${onnxruntime_customopregistration_test_LIBS}
             DEPENDS ${all_dependencies}
     )
+
+    if (CMAKE_SYSTEM_NAME STREQUAL "iOS")
```

Comment:
for the xctest targets, the TARGET_FILE_DIR happens to be different for each target. we can copy the test data in a more general way but this is a simple fix for now.